framework-cloud/framework-openshift-templates/src/main/java/org/kie/cloud/openshift/scenario/HACepScenarioImpl.java

```diff
@@ -125,16 +135,31 @@ protected void deployKieDeployments() {
             throw new RuntimeException("File with HACEP role binding can not be found: " +
                                                roleBindingYamlFile.getAbsolutePath());
         }
+        logger.info("Creating role for HACEP from file: {}", roleYamlFile.getAbsolutePath());
         project.createResourcesFromYamlAsAdmin(roleYamlFile.getAbsolutePath());
+        logger.info("Creating service account for HACEP from file: {}", serviceAccountYamlFile.getAbsolutePath());
         project.createResourcesFromYamlAsAdmin(serviceAccountYamlFile.getAbsolutePath());
+        logger.info("Creating role binding for HACEP from file: {}", roleBindingYamlFile.getAbsolutePath());
         project.createResourcesFromYamlAsAdmin(roleBindingYamlFile.getAbsolutePath());
 
-        project.createResourcesFromYamlAsAdmin(OpenShiftConstants.getHaCepResourcesList());
+        final String imageURL = buildHACEPImage();
+        final File haCepDeploymentYamlFile = new File(haCepSourcesDir, SOURCES_FILE_HACEP_DEPLOYMENT);
+        final String haCepDeploymentYaml = patchHACEPDeploymentFile(haCepDeploymentYamlFile, imageURL);
+        logger.info("Creating HACEP deployment");
+        project.createResourcesFromYamlStringAsAdmin(haCepDeploymentYaml);
+
+        final File haCepService = new File(haCepSourcesDir, SOURCES_FILE_HACEP_SERVICE);
+        logger.info("Creating HACEP service from file: {}", haCepService.getAbsolutePath());
+        project.createResourcesFromYamlAsAdmin(haCepService.getAbsolutePath());
+
         haCepDeployment = new HACepDeploymentImpl(project);
+
+        logger.info("Exposing HACEP service as route");
         project.runOcCommandAsAdmin("expose", "service",
                                     ((HACepDeploymentImpl) haCepDeployment).getServiceName());
         haCepDeployment.waitForScale();
         try {
+            logger.info("Waiting 30 seconds for system to wake up (RHPAM-2380)");
             Thread.sleep(30000); // RHPAM-2380
```

Comment:
For what exactly we are waiting there? Could be there used SimpleWaiter or [waitUntilPodsAreReadyAndRunning](https://github.com/kiegroup/kie-cloud-tests/blob/master/framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/deployment/OpenShiftDeployment.java#L160)

Reply:
Seems that the old issue which caused this delay is no longer valid. Waiting can be removed.